ios/PurchasesHybridCommon/PurchasesHybridCommonIntegrationTests/StoreKitIntegrationTests.swift

```diff
@@ -17,10 +17,77 @@ import XCTest
 
 class StoreKit2IntegrationTests: StoreKit1IntegrationTests {
 
-    override class var storeKit2Setting: StoreKit2Setting { return .enabledForCompatibleDevices }
+    override class var storeKitVersion: StoreKitVersion { return .storeKit2 }
 
 }
 
+class StoreKit2ObserverModeIntegrationTests: BaseIntegrationTests {
+
+    override class var purchasesAreCompletedBy: PurchasesAreCompletedBy {
+        return .myApp
+    }
+
+    override class var storeKitVersion: StoreKitVersion { return .storeKit2 }
+
+    private var testSession: SKTestSession!
+
+    override func setUp() async throws {
```

Comment:
Should we move this up to `BaseIntegrationTests` to avoid duplication?

Reply:
Done! 😄